public/js/views/management/pay-methods.jsx

```diff
@@ -35,6 +40,12 @@ export default class PayMethods extends Component {
   }
 
   render() {
+
+    var isDeleteDisabled = !this.props.payMethods ||
```

Comment:
yeah, I think it's probably safer for other browsers to flatten it. Cool, if babel does it I'm not worried :)